typed-racket-more/typed/images/icons.rkt

```diff
@@ -0,0 +1,212 @@
+#lang typed/racket/gui
+
+(require "logos.rkt" (submod "logos.rkt" digitama))
+(require typed/racket/draw)
+
+(require/typed
+ images/private/deep-flomap
+ [#:opaque Flomap-Material deep-flomap-material-value?])
```

Comment:
Yes, so `deep-flomap` is.
But here is `deep-flomap-material-value` which is left untyped, and lots of values of its instances are undocumented.